.github/workflows/run-compile-requirements.yml

```diff
@@ -11,20 +11,20 @@ jobs:
   call-compile-requirements-linux:
     uses: ./.github/workflows/compile-requirements.yml
     with:
-      requirements_files: '["base.in dev.in gui-dev.in gui.in linters.in"]'
+      requirements_files: '["base.in base.universal2.in dev.in gui-dev.in gui.in linters.in"]'
       os: '["ubuntu-latest"]'
       python: '["3.8", "3.9", "3.10", "3.11"]'
 
   call-compile-requirements-windows:
     uses: ./.github/workflows/compile-requirements.yml
     with:
-      requirements_files: '["base.in dev.in gui-dev.in gui.in linters.in"]'
```

Comment:
Shouldn't the `base.universal2.in` package only be specified for macos?

Reply:
Yes - nice catch. It has no effect on the other jobs, and is thus redundant. Will remove.